timesheet_helpdesk_ticket/models/helpdesk_ticket_team.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models, fields, api
+
+class HelpdeskTicketTeam(models.Model):
+    _inherit = 'helpdesk.ticket.team'
+
+    archive = fields.Boolean('Archive', related='active', default=True, store=True, readonly=False)    
+    
+    allow_timesheet = fields.Boolean(
+        string="Allow Timesheet",
+    )
+    default_project = fields.Many2one(
```

Comment:
Separate project logic from the helpdesk timesheet logic

Reply:
Changed in #121